src/services/backup.ts

```diff
@@ -0,0 +1,48 @@
+import child_process from 'child_process'
+import fs from 'fs'
+import { BACKUP_PATH } from '..'
+import { DB_URL } from './env'
+
+const exec = (command: string) => new Promise((resolve: any, reject) => child_process.exec(command, (err: child_process.ExecException | null, stdout: string, stderr: string) => {
+    if (err || stderr) {
+        console.log('exec command error ', command)
+        reject(err || stderr)
+    }
+    resolve()
+}))
+
+export const getBackupFiles = (): Promise<string[]> => new Promise((resolve, reject) => fs.readdir(BACKUP_PATH, (err, files) => {
```

Comment:
there is fs promise api
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api